b2share/modules/records/serializers/schemas/datacite.py

```diff
@@ -119,6 +119,10 @@ def get_contributors(self, obj):
                 for c in obj['metadata'].get('contributors', [])]
 
     def get_descriptions(self, obj):
+        datacite_v3_description_types = set([
+            "Abstract", "Methods", "SeriesInformation", "TableOfContents"])
+        convert = lambda dt: dt if dt in datacite_v3_description_types else "Other"
```

Comment:
The lambda is called only in one place. IMHO we could as well inline it directly in the list comprehension. Calling functions in python is expensive and there is not much difference in code readability.